deploy/ccm/ccm.properties

```diff
@@ -28,7 +28,7 @@
       "/var/cores/px_info.log",
       "/var/cores/px_patch_fs.log"
     ],
-    "phonehome_hour_range": 8760,
+    "phonehome_hour_range": 744,
```

Comment:
@jrivera-px / @zoxpx is there some way to flag to the larger org that this is changing? 
I have a hard time believing that trying to extract the last year of logs was crucial to anyone's workflow, but it's probably best to double check